WanderLost/WanderLost/Client/Components/EmptyCard.razor

```diff
@@ -0,0 +1,16 @@
+﻿<div class="card">
+  <div class="card__title">@(MerchantName) - @(Region)</div>
```

Comment:
True, will change

Reply:
On the other hand - I don't think `MerchantName` is needed since the merchant is the same for the Region and the user will look for the region when filling in the info (and will probs not even know / care about the merchant name)

Reply:
Let's keep it.

Normally you're correct, but there's in fact two merchants that spawn in Luterra and if I'm not mistaken they occasionally spawn at the same time

Reply:
Oh, did not know that, sure, kept it with the `MerchantName`